src/components/apps/dungeon/home.tsx

```diff
@@ -196,8 +196,8 @@ export function DungeonApp()
 
 
     const handleBetChange = (selected : BetSize) => {
-        
-        if (!isMuted) wagerSelectAudio.play()
+        wagerSelectAudio.volume=volume/100
+        if (muteState!==1) wagerSelectAudio.play()
```

Comment:
in general please put spaces between logical/mathematical operators:
muteState !== 1

volume = volume / 100

etc

I wont leave a comment by each one but please go through

Reply:
got it